airbyte/helm/airbyte/values.yaml.tpl

```diff
@@ -41,20 +41,12 @@ private:
 
 {{ $minioNamespace := namespace "minio" }}
 
-{{ if not .Values.postgresDisabled }}
-airbyte:
-  postgresql:
```

Comment:
we should probably keep this to be safe.  I've had weird behavior with helm values before

Reply:
It's on line 44-46 and 49 now. Needed to move it to another location since the value is in a different spot. Since the location has changed existing values in the `values.yaml` in installation repos won't get touched. I've tested the upgrade and nothing strange happened to the values file.